lib/providers/gcm.js

```diff
@@ -68,7 +68,7 @@ GcmProvider.prototype._createMessage = function(notification) {
   });
 
   Object.keys(notification).forEach(function (key) {
-    if (notification[key] != null) {
+    if (notification[key]) {
```

Comment:
I think this is actually a breaking change, as `{ customKey: false }` is ignored now, which may be a problem. Could you please rework the implementation to ignore only `null` and `undefined`?